src/components/ui/AboutDetails.tsx

```diff
@@ -1,16 +1,16 @@
-import {
-  BedOutlined,
-  Light,
-  ShowerOutlined,
-  TableRestaurant,
-} from '@mui/icons-material'
-import { Box, Grid, Paper, Skeleton, Typography } from '@mui/material'
-import BookingDetails from './BookingDetails'
-import { Room } from '../../types'
+import { BedOutlined, Wifi } from '@mui/icons-material';
+import PoolOutlinedIcon from '@mui/icons-material/PoolOutlined';
+import SmokeFreeOutlinedIcon from '@mui/icons-material/SmokeFreeOutlined';
+import RestaurantIcon from '@mui/icons-material/Restaurant';
+import FitnessCenterIcon from '@mui/icons-material/FitnessCenter';
+import WineBarIcon from '@mui/icons-material/WineBar';

```

Comment:
Group all of these imports into one single import statement from `@mui/icons-material` , like the first import.

Reply:
Done